keyboards/mechwild/mercutio/keymaps/jonavin/keymap.c

```diff
@@ -56,30 +64,90 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   	KC_TRNS, KC_TRNS, KC_TRNS,          KC_BSPC, KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS,          KC_TRNS )
 };
 
+/*  These are needed whether encoder function is enabled or not when ENCFUNC keycode is pressed.
+    Defaults never changes if no encoder present to change it
+*/
+const uint16_t fkeycodes[] = { // list of key codes that will be scrollled through by encoder
+    KC_TASK, KC_INS, KC_DEL, KC_PSCR, KC_SCLN, KC_PAUS, KC_CAD, KC_AF4, KC_MEDIA_PLAY_PAUSE
+};
+const char* fkeydesc[] =    { // list of desc to be shown on LCD max 5 chars will be shown
```

Comment:
Best to use progmem, whenever possible for OLED rendering.  I think you need: 

```suggestion
static const PROGMEM char* fkeydesc[] =    { // list of desc to be shown on LCD max 5 chars will be shown
```

Reply:
The string array is not used to directly render OLED but used in a oled_write() call and dynamically changes. The whole array would have to be loaded into ram anyways. Would this not just complicate the need to read from PROGMEM and had no operational ram savings?